README.md

```diff
@@ -64,30 +64,44 @@ should not require general ternary matches on the destination address; the
 constraint documents this intend and let's us catch accidental ternary matches
```

Comment:
Just noticed this spelling error, if this is a quick fix can you add it to this PR?

Reply:
I assume you are referring to "intend" -> "intent"? Done.